packages/fiori/src/ShellBar.js

```diff
@@ -483,6 +483,10 @@ class ShellBar extends UI5Element {
 		};
 	}
 
+	getStaticAreaItemStableDomRef() {
+		return "shellbar-static";
```

Comment:
It might be weird to have 2 shellbar static area items with same stable dom ref, but I think that's fine.

Reply:
Every ```ui5-shellbar``` only has knowledge about the static area item that is related to it(and not to the all of the static area items). It is ok to have 2 shellbar static area items with same stable dom ref